rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -230,43 +265,47 @@ rosbag2_storage::BagMetadata SqliteStorage::get_metadata()
 {
   rosbag2_storage::BagMetadata metadata;
   metadata.storage_identifier = "sqlite3";
-  metadata.relative_file_paths = {database_name_};
+  metadata.relative_file_paths = {database_files_};
```

Comment:
You don't need the curly braces here. In `{database_name_}` it is there to build in-place a `std::vector<std::string>` but this already has the right type.